plugins/modules/packaging/os/homebrew_cask.py

```diff
@@ -454,18 +465,50 @@ def _current_cask_is_installed(self):
             self.message = 'Invalid cask: {0}.'.format(self.current_cask)
             raise HomebrewCaskException(self.message)
 
-        cmd = [
-            "{brew_path}".format(brew_path=self.brew_path),
-            "cask",
-            "list",
-            self.current_cask
-        ]
+        if self._brew_cask_command_is_deprecated():
+            base_opts = [
+                "{brew_path}".format(brew_path=self.brew_path),
+                "list",
+                "--cask",
+            ]
+        else:
+            base_opts = [
+                "{brew_path}".format(brew_path=self.brew_path),
```

Comment:
Same here.

I guess `base_opts` can also be done in one line after these changes.